fearless/Common/Network/Subscan/Data/PriceData.swift

```diff
@@ -1,13 +1,13 @@
 import Foundation
 
-struct PriceData: Codable, Equatable {
```

Comment:
Now we a caching this entity. So we need to implement both Codable and Equatable